resources/store/actions.js

```diff
@@ -248,5 +248,12 @@ module.exports = {
 	 */
 	updatePublishStatus: function ( context, publishStatus ) {
 		context.commit( 'setPublishStatus', publishStatus );
+	},
+
+	addCustomTag: function ( context, tag ) {
```

Comment:
One piece of the puzzle is missing in `actions.js` – the API call to the database should only contain suggested tags, not custom tags, while the statements sent to the wbsetclaim endpoint should contain all confirmed tags regardless of origin.

Reply:
Good catch. I'll fix this – once this PR is merged, I'll work on getting support for the custom edit tag enabled again as well.

Reply:
See the model change above – all Suggestion model objects will now have a `custom` property which defaults to `false`; Suggestions created through this action will have theirs set to true. This property is used to filter out the custom tags from the review batch on submit. Pretty similar to how we handle it in the "classic" version.